luna/pretrained_models/alexnet.py

```diff
@@ -2,13 +2,18 @@
 A Keras Implementation for Alexnet from
 https://github.com/heuritech/convnets-keras/blob/master/convnetskeras/convnets.py
 """
+from  pathlib import Path
 from keras import backend as K
 from keras.engine import Layer
 from keras.models import Model
 from keras.layers import (Activation, Dense, Dropout, Flatten, Input,
                           concatenate, Lambda)
-from keras.layers.convolutional import (Convolution2D, MaxPooling2D,
+from keras.layers.convolutional import (MaxPooling2D,
                                         ZeroPadding2D)
+from keras.utils.data_utils import get_file
+from tensorflow.keras.layers import Conv2D
+
+
```

Comment:
In general there is a vide variety of blank lines after the import in our files. This should be unified (probably to two).